File: ePerscription.Domain/Entities/PrescriptionItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ePerscription.Domain.Entities
{
    public class PrescriptionItem
    {
        public int Id { get; set; }
        public int PrescriptionId { get; set; }
        public int DrugId { get; set; }
        public string Dose { get; set; }        // e.g. "500 mg"
        public string Form { get; set; }        // e.g. "Tablet"
        public string Frequency { get; set; }   // e.g. "BID"
        public string Duration { get; set; }    // e.g. "7 days"
        public string Instructions { get; set; }

        // Navigation
        public Prescription Prescription { get; set; }
        public Drug Drug { get; set; }
    }
}

File: frontend/eprescription/src/app/features/create-prescription/models/PrescriptionItem.ts
export interface PrescriptionItem {
    id: number;
    Dose: string;// e.g. "500 mg"
    prescriptionId: number;
    Form: string; // e.g. "Tablet"
    drugId: number;
    drugName:string;
    frequency: string;    // e.g. "BID"  
    duration: string;// e.g. "7 days"
    instructions: string; //e.g. after meal
}